flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/builtin/NtPidTable.java

```diff
@@ -0,0 +1,85 @@
+package com.flash3388.frc.nt.builtin;
+
+import com.beans.observables.ObservableDoubleValue;
+import com.flash3388.flashlib.robot.control.PidController;
+import com.flash3388.frc.nt.NtTable;
+import com.flash3388.frc.nt.beans.NtDoubleProperty;
+
+public class NtPidTable {
+    private static final String PID_TABLE_NAME = "Pid";
+    private static final String KP_ENTRY_NAME = "Kp";
+    private static final String KI_ENTRY_NAME = "Ki";
+    private static final String KD_ENTRY_NAME = "Kd";
+    private static final String KF_ENTRY_NAME = "Kf";
+    private static final String PID_INPUT_ENTRY_NAME = "Input";
+    private final NtTable mTable;
+
+    public NtPidTable(double kP, double kI, double kD, double kF, ObservableDoubleValue pidInput) {
+        mTable = new NtTable.Builder(PID_TABLE_NAME)
+                .addDoubleEntry(KP_ENTRY_NAME, 0)
+                .addDoubleEntry(KI_ENTRY_NAME, 0)
+                .addDoubleEntry(KD_ENTRY_NAME, 0)
+                .addDoubleEntry(KD_ENTRY_NAME, 0)
+                .addDoubleEntry(KF_ENTRY_NAME, 0)
+                .addDoubleEntry(PID_INPUT_ENTRY_NAME, 0)
+                .build();
+
+        setPIDFValues(kP, kI, kD, kF);
+        addPidInputListener(pidInput);
+    }
+
+    public static NtPidTable onlyPotential(double kP, ObservableDoubleValue pidInput) {
```

Comment:
The `only` prefix  sounds weird..... remove maybe?

Reply:
so just put P and PD and so on? also, I used the capital letter conventions even though it's `Pid` everywhere else. I just felt that capital letters are more appropriate in this context of the constants themselves.

Reply:
You can use `create` prefix because it's a factory method. But maybe `withP` and such.

Reply:
yeah ok